os/env.go

```diff
@@ -0,0 +1,32 @@
+package os
+
+import "os"
+
+// EnvironmentRepository ...
+type EnvironmentRepository interface {
+	SetVariable(key string, value string) error
+	UnsetVariable(key string) error
+	ListVariable() []string
+}
+
+type defaultEnvironmentRepository struct{}
+
+// NewEnvironmentRepository ...
+func NewEnvironmentRepository() EnvironmentRepository {
+	return defaultEnvironmentRepository{}
+}
+
+// SetVariable ...
+func (m defaultEnvironmentRepository) SetVariable(key string, value string) error {
```

Comment:
Is the receiver intentionally called `m`?

Reply:
Good catch, I'll rename it. Do you have a suggestion?

Reply:
_r_ 😁